plugins/Actions/Archiver.php

```diff
@@ -200,7 +200,6 @@ protected function archiveDayActions($rankingQueryLimit, array $actionTypes, $in
         $rankingQuery = false;
         if ($rankingQueryLimit > 0) {
             $rankingQuery = new RankingQuery($rankingQueryLimit);
-            $rankingQuery->setOthersLabel(DataTable::LABEL_SUMMARY_ROW);
```

Comment:
I suppose we will need to do the same for Custom Reports where we currently use `-1`. We could simply change it depending on the Matomo version.